test/8bit_lds_test.c

```diff
@@ -0,0 +1,229 @@
+#include <criterion/criterion.h>
+#include <criterion/new/assert.h>
+#include <criterion/parameterized.h>
+
+#include "fixtures/cpu_mmu.h"
+#include "common/util.h"
+
+typedef struct LD_8bit_TestParams {
+    uint8_t opcode;
+    uint8_t lhs_val;
+    uint8_t rhs_val;
+    uint8_t l_dword_reg;
+    uint8_t l_word_offset;
+    uint8_t r_dword_reg;
+    uint8_t r_word_offset;
+    uint8_t expected;
+} LD_8bit_TestParams;
```

Comment:
Technically, the structs & typedef names should be using `PascalCase` to comply with the naming conventions of the rest of the codebase.